web/templates/dropdown_list.hbs

```diff
@@ -0,0 +1,13 @@
+{{#with item}}
+    <li class="list-item" role="presentation" data-unique-id="{{unique_id}}" data-name="{{name}}" tabindex="0">
+        <a role="menuitem">
+            {{#if stream}}
+                {{> inline_decorated_stream_name stream=stream show_colored_icon=true}}
+            {{else if is_direct_message}}
+                <i class="zulip-icon zulip-icon-users stream-privacy-type-icon"></i> {{name}}
+            {{else}}
+                {{name}}
+            {{/if}}
+        </a>
+    </li>
+{{/with}}
```

Comment:
Let's start a #frontend conversation about whether we want to create a `web/templates/components` directory for this sort of thing. Can definitely be a follow-up as it's just about moving around code.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/dropdown.20list.20handlebars